spec/presenters/step_by_step_page_presenter_spec.rb

```diff
@@ -40,4 +39,46 @@
       expect(subject).to eq(summary_with_links_checked)
     end
   end
+
+  describe "#broken_link_status_text" do
+    let(:step) { create(:step) }
+    let(:step_nav) { create(:step_by_step_page, slug: 'step-by-step-with-one-step', steps: [step]) }
+    let(:broken_link) {
+      {
+        "uri": "https://www.gov.uk/404",
+        "status": "broken",
+        "checked": "2017-04-12T16:30:39Z",
+        "errors": [
+          "Received 404 response from the server."
+        ],
+        "warnings": [],
+        "problem_summary": "404 error (page not found)",
+        "suggested_fix": ""
+      }
+    }
+
+    subject { described_class.new(step_nav) }
+
+    before do
+      allow(step).to receive(:link_report?).and_return(true)
+    end
+
+    it "should use singular 'link' if only one link is broken" do
```

Comment:
I think this test is named incorrectly.  Something like `should pluralise 'links' if no links are broken`.

Reply:
Whoops, well spotted. A rogue copy & paste. Will fix 👍

Reply:
That's fine by me